### ISSUE TITLE
model is not learning on my dataset

### ISSUE THREAD

**Issue by widedh** [fizyr/keras-retinanet#1014]
Hi,
I see that the model is not learning on my dataset. The loss is stuck at these values from start till end. 
Help please.
1000/1000 [==============================] - 202s 202ms/step - loss: 2.7603 - regression_loss: 2.1831 - classification_loss: 0.5771
1000/1000 [==============================] - 199s 199ms/step - loss: 2.7278 - regression_loss: 2.1850 - classification_loss: 0.5428
1000/1000 [==============================] - 187s 187ms/step - loss: 2.7454 - regression_loss: 2.1994 - classification_loss: 0.5460
1000/1000 [==============================] - 182s 182ms/step - loss: 2.7257 - regression_loss: 2.1847 - classification_loss: 0.5410
1000/1000 [==============================] - 180s 180ms/step - loss: 2.7213 - regression_loss: 2.1979 - classification_loss: 0.5234
1000/1000 [==============================] - 181s 181ms/step - loss: 2.7545 - regression_loss: 2.1976 - classification_loss: 0.5570
1000/1000 [==============================] - 180s 180ms/step - loss: 2.7312 - regression_loss: 2.1964 - classification_loss: 0.5349
1000/1000 [==============================] - 180s 180ms/step - loss: 2.7624 - regression_loss: 2.1731 - classification_loss: 0.5893

**Comment by martinzlocha** [comment]
Try to check that your bounding boxes look good in debug.py

**Comment by widedh** [comment]
I solved the issue by restarting the train with the default weights
